draft-ietf-scitt-architecture.md

```diff
@@ -158,59 +160,59 @@ Statement:
 : any serializable information about an Artifact. To help interpretation of Statements, they must be tagged with a media type (as specified in {{RFC6838}}).
 For example, a statement may represent a Software Bill Of Materials (SBOM) that lists the ingredients of a software Artifact, or some endorsement or attestation about an Artifact.
 
-Claim:
+Signed Statement:
 
-: an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Claims are encoded as COSE signed objects; the payload of the COSE structure contains the Statement.
+: an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Signed Statements are encoded as COSE signed objects; the payload of the COSE structure contains the issued Statement.
 
 Issuer:
 
-: an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer or an endorser.
+: an entity that creates Signed Statements about software Artifacts in the supply chain. An Issuer may be the owner or author of software Artifacts, or an independent third party such as a reviewer or an endorser.
 
 Envelope:
 
-: the metadata added to the Statement by the Issuer to make it a Claim.
-It contains the identity of the Issuer and other information to help Verifiers identify the Artifact referred in the Statement. A Claim binds the Envelope to the Statement. In COSE, the Envelope consists of protected headers.
+: metadata and an Issuer's signature is added to a Statement via COSE envelope by the Issuer to produce a Signed Statement.
```

Comment:
> metadata and an Issuer's signature is added to a Statement via COSE envelope 

metadata and an Issuer's signature is added to a Statement via _**a**_ COSE envelope